packages/components/src/pages/Quickstart.vue

```diff
@@ -0,0 +1,571 @@
+            Run tests to create AppMaps
+          </button>
+          <div
+            class="qs-loader"
+            v-if="isActionRunning"
+            data-process="5 AppMaps created"
+          >
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+          </div>
+        </div>
+        <div class="qs-step__success" v-if="step3Completed">
+          <span class="qs-step__success-title">
+            <SuccessIcon class="qs-step__success-icon" />
+            AppMaps recorded
+          </span>
+          <button
+            type="button"
+            class="qs-step__success-next-step qs-button"
+            @click="nextStep"
+          >
+            Next step : View AppMaps ->
+          </button>
+        </div>
+      </section>
+    </div>
+    <div class="qs-help">
+      <HelpIcon class="qs-help__icon" />
+      <div class="qs-help__text">
+        <p>Need help or want to give feedback?</p>
+        <a
+          href="https://appland.com/company/contact-us"
+          target="_blank"
+          rel="noopener noreferrer"
+          >Contact an AppLand developer directly.</a
+        >
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import HelpIcon from '@/assets/quickstart/help.svg';
+import StepLeftIcon from '@/assets/quickstart/arrow-left.svg';
+import StepRightIcon from '@/assets/quickstart/arrow-right.svg';
+import SuccessIcon from '@/assets/quickstart/success.svg';
+
+export const Steps = {
+  INSTALL_AGENT: 1,
+  CONFIGURE_APPMAP: 2,
+  RECORD_APPMAPS: 3,
+};
+
+export default {
+  name: 'Quickstart',
+
+  components: {
+    HelpIcon,
+    StepLeftIcon,
+    StepRightIcon,
+    SuccessIcon,
+  },
+
+  props: {
+    language: {
+      type: String,
+      default: null,
+    },
+    initialStep: {
+      type: Number,
+      default: 1,
+    },
+    completedSteps: {
+      type: Array,
+      default: () => [],
+    },
+    onAction: {
+      type: Function,
+    },
+    onStep: {
+      type: Function,
+    },
+  },
+
+  data() {
+    return {
+      currentStep: this.initialStep,
+      step1Completed: this.completedSteps.includes(1),
+      step2Completed: this.completedSteps.includes(2),
+      step3Completed: this.completedSteps.includes(3),
+      step4Completed: this.completedSteps.includes(4),
+      isActionRunning: false,
+    };
+  },
+
+  watch: {
+    currentStep: {
+      handler() {
+        if (typeof this.onStep === 'function') {
+          this.onStep(this.currentStep);
+        }
+      },
+    },
+  },
+
+  computed: {
+    stepsCount() {
+      return Object.keys(Steps).length;
+    },
+    languages() {
+      const languages = {
+        ruby: 'Ruby',
+        java: 'Java',
+        python: 'Python',
+      };
+
+      if (Object.keys(languages).includes(this.language)) {
+        languages[this.language] += ' (detected)';
```

Comment:
`(detected)` should only be appended to the language if passed by prop. If the user selects another language from the dropdown, it shouldn't display `(detected)`.